bundles/framework/admin-announcements/view/AnnouncementsFormMapperHandler.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { StateHandler, controllerMixin, Messaging } from 'oskari-ui/util';
+import { announcementsCalls } from './AnnouncementsCalls';
+import { Message } from 'oskari-ui';
+
+/*
+Handler for admin-announcements forms.
+*/
+
+const getMessage = (key, args) => <Message messageKey={key} messageArgs={args} bundleKey='admin-announcements' />;
+
+class ViewHandler extends StateHandler {
+    constructor (instance) {
+        super();
+        this.instance = instance;
+        this.sandbox = instance.getSandbox();
```

Comment:
this.sandbox isn't used in this file. Can it be removed?

Reply:
removed